samples/bot-all-cards/csharp/README.md

```diff
@@ -0,0 +1,151 @@
+---
+page_type: sample
+description: Type of Cards.
+products:
+- dotnet
+- office-teams
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "21/10/2022 07:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-all-cards-csharp
+
+---
+
+# Using Cards
+
+Bot Framework v4 using cards bot sample
```

Comment:
Wrong summary and heading, please check NodeJS PR>

Reply:
Fixed